### PR TITLE
Fix incorrect message formatting + references.

### PR DESCRIPTION
**Pull request by amine3**

### CODE REVIEW

File: agent/metasploit_agent.py [hunk 1]
```diff
@@ -28,6 +29,12 @@
 MODULE_TIMEOUT = 300
 VULNERABLE_STATUSES = ["vulnerable", "appears"]
 METASPLOIT_AGENT_KEY = b"agent_metasploit_asset"
+TOKEN_REPLACE = "REPLACEME"
+REFERENCES = {
+    "CVE": f"https://nvd.nist.gov/vuln/detail/CVE-{TOKEN_REPLACE}",
+    "CWE": f"https://cwe.mitre.org/data/definitions/{TOKEN_REPLACE}.html",
+    "EDB": f"https://www.exploit-db.com/exploits/{TOKEN_REPLACE}",
+}
```
Comment (3asm) [inline agent/metasploit_agent.py:37]: I don't like the way these are replaced. I think you can use named format strings to replace the values directly.

File: agent/metasploit_agent.py [hunk 2]
```diff
@@ -122,7 +129,7 @@ def process(self, message: m.Message) -> None:
                 ):
                     technical_detail = f"Using `{module_instance.moduletype}` module `{module_instance.modulename}`\n"
                     technical_detail += f"Target: {vhost}:{rport}\n"
-                    technical_detail += f'Message: \n```{results["message"]}```'
+                    technical_detail += f'Message: \n```\n{results["message"]}\n```'
```
Comment (3asm) [inline agent/metasploit_agent.py:132]:
```suggestion
                    technical_detail += f'Message: \n```shell\n{results["message"]}\n```'
```

**Review by 3asm — CHANGES_REQUESTED** [review]

File: agent/metasploit_agent.py [hunk 3]
```diff
@@ -28,6 +28,11 @@
 MODULE_TIMEOUT = 300
 VULNERABLE_STATUSES = ["vulnerable", "appears"]
 METASPLOIT_AGENT_KEY = b"agent_metasploit_asset"
+REFERENCES = {
+    "CVE": "https://nvd.nist.gov/vuln/detail/CVE-{ID}",
+    "CWE": "https://cwe.mitre.org/data/definitions/{ID}.html",
+    "EDB": "https://www.exploit-db.com/exploits/{ID}",
+}
```
Comment (BlueSquare1) [inline agent/metasploit_agent.py:35]: another common reference is `OSVDB`

File: agent/metasploit_agent.py [hunk 4]
```diff
@@ -28,6 +28,11 @@
 MODULE_TIMEOUT = 300
 VULNERABLE_STATUSES = ["vulnerable", "appears"]
 METASPLOIT_AGENT_KEY = b"agent_metasploit_asset"
+REFERENCES = {
+    "CVE": "https://nvd.nist.gov/vuln/detail/CVE-{ID}",
```
Comment (benyissa) [inline agent/metasploit_agent.py:32]:
```suggestion
    "NVD": "https://nvd.nist.gov/vuln/detail/CVE-{ID}",
```

File: agent/metasploit_agent.py (hunk 3 above)
Comment (amine3) [reply agent/metasploit_agent.py:35]: Deprecated

**Review by 3asm — APPROVED** [review]

**Pull request merged**